Documentation/building/unix/instructions.md

```diff
@@ -72,36 +72,59 @@ and Windows (.vsix) installer files can be built with:
 Commercial installers will be created by this command if the
 `make prepare-external-git-dependencies` command was ran before building.
 
-# Creating .NET 6 NuGet packages
+A .NET 6 Workload installer is also created at
+`Microsoft.NET.Workload.Android-*.pkg`, you can run `make
+create-workload-installers` if you only want to create the .NET 6
+installer.
+
+# Creating a local .NET 6 Workload
+
+`make prepare` provisions a specific build of .NET 5/6 to
+`~/android-toolchain/dotnet`.
 
 Once `make all` or `make jenkins` have completed, you can build the .NET 6
 packages with:
 
     make pack-dotnet
 
-Several `.nupkg` files will be output in `./bin/BuildDebug/nupkgs/`, you
-can use these with a `nuget.config` such as:
+Several `.nupkg` files will be output in `./bin/BuildDebug/nupkgs`,
+but this is only part of the story. Your local
+`~/android-toolchain/dotnet/packs` directory will be populated with a
+local Android "workload" in `packs/Microsoft.Android.Sdk`.
```

Comment:
WRT #5245 we could maybe update this to say something like:
> Your local `~/android-toolchain/dotnet/packs` directory will be populated with a
local host OS specific Android "workload" SDK in `packs/Microsoft.Android.Sdk.$(RID)`

Reply:
I updated the sentence here, so it will make more sense when #5245 is merged.